components/sections/Blog/index.jsx

```diff
@@ -0,0 +1,75 @@
+import { useRef, useState } from "react";
+import articles from "../../../data/articles";
+import BlogItem from "../../cards/BlogItem";
+import ArrowIcon from "../../ui/ArrowIcon";
+import Title from "../../ui/Title";
+import Wrapper from "../../Wrapper";
+
+const Blog = () => {
+  const itemRef = useRef();
+  const itemsContainerRef = useRef();
+  const [offset, setoffset] = useState(0);
```

Comment:
`setoffset` should be written in camel case
```suggestion
  const [offset, setOffset] = useState(0);
```